packages/create-waku/src/helpers/examples.ts

```diff
@@ -0,0 +1,123 @@
+import * as tar from 'tar';
+import { Readable } from 'stream';
+import { pipeline } from 'stream/promises';
+import { type ReadableStream } from 'stream/web';
```

Comment:
We use separate import line for types.
```suggestion
import type { ReadableStream } from 'stream/web';
```